<launch>

    <!-- <node pkg="rtls_uwb_sensor" type="display" name="uwb_node1" output="log" >
       <remap from="/uwb/data" to="/suav_2/uwb/data" />
    </node> -->

    <node pkg="range_localization" type="range_node" name="range_node1" output="screen" >
        <rosparam command="load" file="$(find range_localization)/config/uav.yaml" />
        <param name="vehicle_name"  value="suav_1" />
        <param name="imu_topic" value="/suav_2/dji_osdk_ros/imu" />
        <param name="range_topic"  value="/suav_1/uwb/data" />
        <param name="pressure_topic"  value="/suav_2/dji_osdk_ros/height_above_takeoff" />
        <rosparam param="init_position" >[-4.024, 3.0, 0.50]</rosparam>
    </node>
        
    <!-- <node pkg="rosbag" type="record" name="bag_record1" args="/suav_2/uwb/data /suav_2/imu/data /suav_2/filter/odom /suav_2/dji_osdk_ros/height_above_takeoff /suav_2/dji_osdk_ros/vo_position /suav_2/dji_osdk_ros/imu -o /home/$(env USER)/01.bag"/> -->
    <node pkg="rviz" type="rviz" name="range_node_rviz" args="-d $(find range_localization)/config/config.rviz" />

</launch>

